internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/consumer/ApacheKafkaConsumerAdapter.java

```diff
@@ -149,6 +149,13 @@ private boolean isValidTopicPartition(PubSubTopicPartition pubSubTopicPartition)
         throw new PubSubClientException(
             "Interrupted while waiting for validation of topic-partition: " + pubSubTopicPartition,
             e);
+      } catch (Exception e) {
```

Comment:
Can we only handle Kafka RetriableException here?

Reply:
done